substrate/bin/utils/chain-spec-builder/tests/test.rs

```diff
@@ -24,6 +24,7 @@ use cmd_lib::spawn_with_output;
 use sc_chain_spec::update_code_in_json_chain_spec;
 use serde_json::{from_reader, from_str, Value};
 use staging_chain_spec_builder::ChainSpecBuilder;
+use pretty_assertions::assert_eq;
```

Comment:
Is this needed?

Reply:
Ah, I understand now - `assert_eq` is already used in the file, but you now chose to use one that is (visually) different, whose name happens to be the same.